Wheel is not available on Solaris yet, but we don't need it to build the component anyway.

--- dnspython-2.2.1/setup.cfg
+++ dnspython-2.2.1/setup.cfg
@@ -46,7 +46,7 @@ packages =
     dns.rdtypes.CH
 python_requires = >=3.6
 test_suite = tests
-setup_requires = setuptools>=44; wheel; setuptools_scm[toml]>=3.4.3
+setup_requires = setuptools>=44; setuptools_scm[toml]>=3.4.3
 
 [options.extras_require]
 DOH = httpx>=0.21.1; h2>=4.1.0; requests; requests-toolbelt
